loom/common/utils/utTypes.h

```diff
@@ -1393,7 +1393,7 @@ class utHashTable
 
 
     // Find and cache key
-    Value *get(const Key& key)
+    Value *get(const Key& key) const
```

Comment:
Is this an appropriate constification? Can't get lead to mutation? Or is it only mutation of value not the actual hash table state?